plugins/lsp/fidget.nix

```diff
@@ -551,11 +543,11 @@ in
 
         floatPrecision = helpers.defaultNullOpts.mkNullable (
           with types; numbers.between 0.0 1.0
-        ) "0.01" "Limit the number of decimals displayed for floats.";
+        ) 1.0e-2 "Limit the number of decimals displayed for floats.";
```

Comment:
stylix is changing this to exponent notation, but I guess that doesn't matter since toPretty can still print it however it wants.

Reply:
Ok